core/test/test_container.cpp

```diff
@@ -611,7 +611,7 @@ TEST(Task, reuse) {
 
 	try {
 		configure(t);
-		EXPECT_TRUE(t.plan(1));
+		EXPECT_TRUE(t.plan(1).val == moveit_msgs::MoveItErrorCodes::SUCCESS);
```

Comment:
I would strongly prefer an error code wrapper class to keep the current boolean check for success.

Ideally we would migrate the `MoveItErrorCode` class you refer to to `moveit_core` (where it should have been to start with...). Alternatively we could copy the class here.